torch/csrc/autograd/custom_function.h

```diff
@@ -98,6 +99,12 @@ struct TORCH_API Function {
   template <typename X = T, typename... Args>
   static auto apply(Args&&... args)
       -> std::enable_if_t<std::is_same_v<X, T>, forward_t<X, Args...>>;
+
+  // Flag overriden by user to enable compiled autograd tracing for custom
+  // function <T>. Before you can set this flag to true, you need to ensure
+  // that all additional static variables used in the forward and backward
+  // are captured in compiled_args and apply_wih_saved.
+  static constexpr bool is_traceable = false;
```

Comment:
Please update the comment bloc above the class that we use as documentation to mention this flag.